app/components-react/hooks.ts

```diff
@@ -89,10 +89,17 @@ export function useFormState<T extends object>(initializer: T | (() => T)): TUse
   // create a reference to the last actual state
   const stateRef = useRef(s);
 
+  // use isDestroyed flag to prevent updating state on destroyed components
```

Comment:
I this mainly a performance optimization?

Reply:
This is a quick fix for the case when a finished xhr request executes a callback that changes the component's state but the component is destroyed. React write errors in the console in that case that cause the failure in automated tests.

For example, in FB tests we click `Confrm & GoLive` before the list of FB groups has been loaded.